package.json

```diff
@@ -20,7 +20,8 @@
   },
   "homepage": "https://github.com/wikimedia/grunt-stylelint",
   "dependencies": {
-      "stylelint": "^7.0.0"
+    "chalk": "1.1.3",
+    "stylelint": "7.7.1"
```

Comment:
Why did you remove the caret `^`?

FWIW: This should probably now use the tilde `~` feature for semantic versioning, previously it was the caret `^` but as of stylelint v8.0.0 the recommendation will be the the tilde `~`